### PR TITLE
Item Description Cleanup Project: Vehicle Parts

### PR DESCRIPTION
**Pull request by FoolsGold45**
#### Summary
None

#### Purpose of change
Fixing spelling and grammar issues in item descriptions. This pull request focuses on vehicle parts.

#### Describe the solution
Fixed awkward wording and inconsistent descriptions found in item descriptions in these categories.

#### Describe alternatives you've considered
The goal was to fix the existing descriptions, not to add new meaning to them outside of select circumstances where I felt the player experience could be improved by better explaining an item's in-game properties and applications. If I've overstepped this purpose in any descriptions, I don't mind changing them back.

### CODE REVIEW

File: data/json/items/vehicle/armor.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "type": "GENERIC",
     "category": "veh_parts",
     "name": { "str": "shock absorber" },
-    "description": "This makeshift combination of springs and scrap, when attached to a vehicle section, protects that section from impacts.  The springs can absorb a surprising amount of damage.",
+    "description": "A makeshift combination of springs and scrap that protects a particular section of a vehicle from impacts.  The springs can absorb a surprising amount of damage.",
```
Comment (Maleclypse) [inline data/json/items/vehicle/armor.json:23]:
```suggestion
    "description": "A makeshift combination of springs and scrap that protects a particular section of a vehicle from indirect damage from impacts.  The springs can absorb a surprising amount of damage.",
```
There's probably a better way to say this but I can't think of it right now.

**Review by I-am-Erk — APPROVED** [review]
lgtm, will merge when tests done unless someone gets it before me

File: data/json/items/vehicle/engineering.json [hunk 2]
```diff
@@ -47,8 +47,8 @@
     "symbol": "&",
     "name": { "str": "rockwheel" },
     "color": "dark_gray",
-    "description": "A large and heavy jagged metal disc to dig trenches.",
-    "weight": "595 kg",
+    "description": "A large, heavy, jagged metal disc for digging trenches.",
+    "weight": "62000 g",
```
Comment (ZhilkinSerg) [inline data/json/items/vehicle/engineering.json:51]: Vehicle drag values changed.  Update values in `data/mods/TEST_DATA/vehicle_drag_test.json` with values printed below:
`{ "trencher": [ 0.520838, 1.386882, 1576.078846, 8506, 8828 ] },`

Comment (FoolsGold45) [reply data/json/items/vehicle/engineering.json:51]: should be all set.

**Pull request merged**